packages/search/src/Search.ts

```diff
@@ -115,6 +115,7 @@ export class Search extends Textfield {
                 @keydown=${this.handleKeydown}
             >
                 <sp-icon-search
+                    size=${this.size}
```

Comment:
How does the size propagate here then?

Reply:
because here we're directly using the `sp-icon-search` instead of a slot.